### PR TITLE
[ENH] Add fit_method option in DTI and DKI CLI

### LINKED ISSUE #2463: DTI RESTORE on the CLI
## Description
Posted at the request for @skoudoro via gitter.

RESTORE is available as a fitting method for DTI; however CLI support is absent. It would be helpful to have a RESTORE fitting option on the CLI as well.

### PR DESCRIPTION
**Pull request by skoudoro**
This PR should fix #2463.  RESTORE is available as a fitting method for DTI; however, CLI support is absent. So this PR goal is to provide RESTORE and other fitting method options.

### CODE REVIEW

File: dipy/workflows/reconst.py [hunk 1]
```diff
@@ -738,6 +745,12 @@ def run(self, input_files, bvalues_files, bvectors_files, mask_files,
         mask_files : string
             Path to the input masks. This path may contain wildcards to use
             multiple masks at once. (default: No mask used)
+        fit_method : string, optional
+            can be one of the following:
+            'WLS' for weighted least squares
+            'LS' or 'OLS' for ordinary least squares
+            'NLLS' for non-linear least-squares
+            'RT' or 'restore' or 'RESTORE' for RESTORE robust tensor fitting
```
Comment (arokem) [inline dipy/workflows/reconst.py:753]: I don't believe that DKI has a RESTORE option.
```suggestion
```

**Review by arokem — COMMENTED** [review]
LGTM! Just a small documentation suggestion for DKI/RESTORE.

**Comment by arokem** [comment]
See [here](https://github.com/dipy/dipy/blob/8fb0bda64c6fd2bbd6ed2ba67e81c1e58d73a108/dipy/reconst/dki.py#L1541) for the options for DKI

File: dipy/workflows/reconst.py (hunk 1 above)
Comment (skoudoro) [reply dipy/workflows/reconst.py]: Good catch and Thanks! I just addressed your comment.

**Comment by skoudoro** [comment]
Thanks for the review @arokem. I addressed your comment so this PR seems ready to go.

**Pull request merged**